utils/utils.go

```diff
@@ -199,6 +210,22 @@ func pathToRegExp(localPath string) string {
 	return "^" + localPath + "$"
 }
 
+func antPathToRegExp(localPath string) string {
+	var SPECIAL_CHARS = []string{".", "^", "$", "+"}
```

Comment:
The go convention does not support camel-casing such as SPECIAL_CHARS.
SPECIAL_CHARS should be changed to **specialChars**.
Tge same goes for the ```wildcardPathToRegExp``` function.